site/content/en/docs/manual/advanced/analytics-and-monitoring/manual-qa.md

```diff
@@ -0,0 +1,204 @@
+---
+title: 'Manual QA and Review'
+linkTitle: 'Manual QA'
+weight: 2
+description: 'Guidelines on evaluating annotation quality in CVAT manually'
+---
+
+In the demanding process of annotation, ensuring accuracy is paramount.
+
+CVAT introduces a specialized **Review mode**, designed to streamline the
+validation of annotations by pinpointing errors or discrepancies in annotation.
+
+> **Note**: The **Review mode** is not applicable for 3D tasks.
+
+See:
+
+- [Review and report issues: review only mode](#review-and-report-issues-review-only-mode)
+  - [Assigning reviewer](#assigning-reviewer)
+  - [Reporting issues](#reporting-issues)
+  - [Quick issue](#quick-issue)
+  - [Assigning corrector](#assigning-corrector)
+  - [Correcting reported issues](#correcting-reported-issues)
+- [Review and report issues: review and correct mode](#review-and-report-issues-review-and-correct-mode)
+- [Issues navigation and interface](#issues-navigation-and-interface)
+  - [Issues tab](#issues-tab)
+  - [Issues workspace](#issues-workspace)
+  - [Issues comments](#issues-comments)
+- [Manual QA complete video tutorial](#manual-qa-complete-video-tutorial)
+
+## Review and report issues: review only mode
+
+Review mode is a user interface (UI) setting where a specialized
+**Issue** tool is available. This tool allows you to identify
+and describe issues with objects or areas within the frame.
+
+> **Note:** While in review mode, all other tools will be hidden.
+
+**Review** mode screen looks like the following:
+
+![Review mode screen](/images/review_mode_screen.jpg)
+
+### Assigning reviewer
+
+> **Note**: Only project owners and maintainers can assign reviewers.
```

Comment:
That is not exactly true.

Reviewers can be assigned by project/task owner/assignee.
And by organization maintainers of course.